packages/core/src/http-middlewares/before/prepare-request.js

```diff
@@ -105,14 +105,15 @@ const finalRequest = req => {
 const prepareRequest = function(req) {
   const input = req.input || {};
 
-  // We will want to use _.defeaultsDeep if one of these nested values ever defaults to true.
+  // We will want to use _.defaultsDeep if one of these nested values ever defaults to true.
   req = _.defaults(req, {
     merge: true,
     removeMissingValuesFrom: {
       params: false,
       body: false
     },
     replace: true, // always replace curlies
+    skipThrowForStatus: false,
```

Comment:
Although `undefined` is falsy too, make sure the new flag is there on the request